### PR TITLE
Allow to pass DB_FILTER env var

### PR DESCRIPTION
**Pull request by PierrickBrun**
This PR allows to use DB_FILTER to customize what databases are available in the server.

This can be very useful in a development context with `DB_FILTER = .*`

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: 10.0/templates/odoo.cfg.tmpl [hunk 1]
```diff
@@ -6,7 +6,7 @@ db_host = {{ .Env.DB_HOST }}
 db_name = {{ .Env.DB_NAME }}
 db_user = {{ .Env.DB_USER }}
 db_password = {{ .Env.DB_PASSWORD }}
-dbfilter = ^{{ default .Env.DB_NAME "odoodb" }}$
+dbfilter = ^{{ default .Env.DB_FILTER .Env.DB_NAME }}$
```
Comment (simahawk) [inline 10.0/templates/odoo.cfg.tmpl:9]: shouldn't you keep `"odoodb"` as default?

Comment (PierrickBrun) [reply 10.0/templates/odoo.cfg.tmpl:9]: Well `db_name` has no default so I expect it to be always populated.

Also I don't see any reason to have `db_name` not included in `dbfilter` (for instance if DB_NAME='db' and DB_FILTER not set)

Comment (simahawk) [reply 10.0/templates/odoo.cfg.tmpl:9]: yeah, you are right. We always have DB_NAME anyway.

**Review by simahawk — APPROVED** [review]
@p-tombez or @yvaucher fine for you?

**Review by yvaucher — APPROVED** [review]

**Pull request merged**